automation/src/test/scala/org/broadinstitute/dsde/firecloud/page/workspaces/methodconfigs/WorkspaceMethodConfigDetailsPage.scala

```diff
@@ -37,14 +38,15 @@ class WorkspaceMethodConfigDetailsPage(namespace: String, name: String, methodCo
   def clickLaunchAnalysis(): Unit = {
     openLaunchAnalysisModalButton.doClick()
     // defensive code to prevents test from failing. after click, expect to find either a Message or Analysis Modal
-    implicit val patienceConfig: PatienceConfig = PatienceConfig(timeout = scaled(Span(10, Seconds)), interval = scaled(Span(3, Seconds)))
-    Try[String](eventually[String] {
-      find(CssSelectorQuery("body.broadinstitute-modal-open")).get.underlying.getTagName
-    }) match {
+    Try(
+      await condition (find(CssSelectorQuery(".broadinstitute-modal-open")).nonEmpty, 5)
+    ) match {
       case Failure(e) =>
-        logger.debug(s"Retrying click [button:${openLaunchAnalysisModalButton.query}]")
-        openLaunchAnalysisModalButton.doClick()
+        logger.warn(s"clickLaunchAnalysis failed. Retrying click [button:${openLaunchAnalysisModalButton.query}]")
+        // try alternative click
+        new Actions(webDriver).moveToElement(openLaunchAnalysisModalButton.query.webElement).click().build().perform();
```

Comment:
Why do we need to do this instead of just clicking the button?

Reply:
I am suspecting chromedriver click has issue with click that fails occasionally.

Reply:
As I've expressed before, I'm highly skeptical of such a bug. I suspect that, if this change improves anything, it will be because of timing changes due to the addition of `moveToElement`. The real solution is to fix the timing in the test.

Alternately, we may have a case where an element is available but then some UI state changes and causes a re-render that makes the element briefly unavailable and then available again by the time the screenshot is taken. We can investigate this by identifying flaky behavior and adding some logging to the UI render method. If we find that this is the case, we should ask for advice from our UI experts.

Reply:
use `doClick`